internal/experiment/generation/containerresources.go

```diff
@@ -64,10 +67,8 @@ func (s *ContainerResourcesSelector) Map(node *yaml.RNode, meta yaml.ResourceMet
 				p := containerResourcesParameter{pnode{
 					meta:      meta,
 					fieldPath: append(path, "[name="+name+"]", "resources"),
+					value:     rl.Value.YNode(),
 				}}
-				if rl != nil {
-					p.value = rl.Value.YNode()
-				}
```

Comment:
so at this point, if rl was nil do we end up with?
```
resources: {}
```

Reply:
Effectively, yes. Technically, I don't set the key because we don't need it.